src/Hello/OpenApi/RawFile.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Hello\OpenApi;
+
+use Exception;
+use Symfony\Component\Yaml\Yaml;
+
+class RawFile
+{
+    /**
+     * We expect translation key strings to look like
+     * "_t__AccountCreate::SUMMARY"
+     */
+    private const TRANSLATE_PREPEND = '_t__';
+
+    /**
+     * We expect markdown key strings to look like
+     * "_md__OpenApi::TAG"
+     */
+    private const MARKDOWN_PREPEND = '_md__';
+
+    /**
+     * Custom key to determine a surface on which a spec
+     * is to be shown.
+     *
+     * e.g.
+     * x-showOn: 'SDK'
+     * x-showOn: 'DOC'
+     * x-showOn: 'NONE'
+     */
+    private const SHOW_ON = 'x-showOn';
+
+    private const PREPENDS = [
+        self::TRANSLATE_PREPEND,
+        self::MARKDOWN_PREPEND,
+    ];
```

Comment:
Let's instead do the inverse. By default we should expect all data to be included in both the documentation and the SDKs, and want special handling for when we _don't_ want to show it on one or the other.

In other words, let's use `x-hideOn` instead of `x-showOn`